include/z64cutscene.h

```diff
@@ -491,6 +496,11 @@ typedef union {
     s32 _words[4];
 } CutsceneCameraPoint; // size = 0x10
 
+/**
+ * #define CS_CAM_POINT(continueFlag, roll, frame, viewAngle, xPos, yPos, zPos, unused) \
+    CMD_BBH(continueFlag, roll, frame), CMD_F(viewAngle), CMD_HH(xPos, yPos), CMD_HH(zPos, unused)
+*/
+
```

Comment:
This is already define in `z64cutscene_commands.h` right? Or is it just here for reference to the struct?

Reply:
oh yeah that was when I wanted to follow a bit how the structs were working with the ``CMD_HH`` macros because I had a small word issue, forgot to remove that lol